### DISCUSSION TITLE
Conversion succeeded but not program.cs created and no config file

### DISCUSSION THREAD

**Discussion by ken-saraf** [CoreWCF/CoreWCF#1189] [Q&A]
Hi,
We ran the upgrade assistant successfully  - according to the upgrade log - but the new project doesn't contain a staring point (program.cs) or a converted web.config file. What might be the cause for this problem? Attached please find the upgrade log.
[upgrade log.txt](https://github.com/CoreWCF/CoreWCF/files/12432583/upgrade.log.txt)

Thank you,
Ken

**Comment by g7ed6e** [comment]
> Hi, We ran the upgrade assistant successfully - according to the upgrade log - but the new project doesn't contain a staring point (program.cs) or a converted web.config file. What might be the cause for this problem? Attached please find the upgrade log. [upgrade log.txt](https://github.com/CoreWCF/CoreWCF/files/12432583/upgrade.log.txt)
> 
> Thank you, Ken

Hi @ken-saraf you should report this issue to https://github.com/dotnet/upgrade-assistant

**Reply by ken-saraf** [reply]
Thank you. Done
